src/pipeline/standard-pipeline.ts

```diff
@@ -88,6 +108,13 @@ export class StandardPipeline extends ObjectRenderer {
       }
       return a.renderSortOrder < b.renderSortOrder ? -1 : 1
     })
+
+    this._sprites.sort((a, b) => {
+      if (a.zIndex !== b.zIndex) {
```

Comment:
Do you see any specific use case for the zIndex?

Reply:
ability to override the standard ordering, this would allow it to function similar to the renderSortOrder of Mesh3D